packages/blade/src/components/FileUpload/FileUpload.web.tsx

```diff
@@ -0,0 +1,367 @@
+          >
+            {label}
+          </FormLabel>
+        ) : null}
+
+        <BaseBox display="flex" flexDirection="column" width="100%">
+          <SelectorLabel
+            componentName={MetaConstants.FileUploadLabel}
+            inputProps={{}}
+            style={{
+              cursor: isDisabled ? 'not-allowed' : 'pointer',
+              ...(isOneFileSelectedWithSingleUpload ? screenReaderStyles : {}),
+            }}
+          >
+            <BaseBox
+              display="flex"
+              flexDirection="column"
+              width="100%"
+              marginBottom={willRenderHintText ? 'spacing.0' : 'spacing.5'}
+            >
+              <StyledFileUploadWrapper
+                isDisabled={isDisabled}
+                isActive={isActive}
+                display="flex"
+                flexDirection="row"
+                justifyContent="center"
+                alignItems="center"
+                borderRadius="medium"
+                borderWidth="thin"
+                borderColor={
+                  isDisabled
+                    ? fileUploadColorTokens.border.disabled
+                    : fileUploadColorTokens.border.default
+                }
+                onDragOver={handleDragOver}
+                onDragLeave={handleDragLeave}
+                onDrop={handleDrop}
+                onClick={() => setIsActive(true)}
+              >
+                <Box
+                  display="flex"
+                  justifyContent="center"
+                  alignItems="center"
+                  flexDirection={{ xs: 'column', s: 'row' }}
+                  gap={makeSize(6)}
+                  padding="spacing.3"
+                >
+                  <Text
+                    color={
+                      isDisabled
+                        ? fileUploadColorTokens.text.disabled
+                        : fileUploadColorTokens.text.default
+                    }
+                  >
+                    Drag files here or{' '}
+                  </Text>
+                  <SelectorInput
+                    id={inputId}
+                    hoverTokens={getFileUploadInputHoverTokens()}
+                    isChecked={false}
+                    isDisabled={isDisabled}
+                    inputProps={{
+                      name,
+                      type: 'file',
+                      onChange: handleInputChange,
+                      multiple: isMultiple,
+                      required: isRequired,
+                      disabled: isDisabled,
+                      accept,
+                      onBlur: () => setIsActive(false),
+                      ...accessibilityProps,
+                    }}
+                    ref={ref}
+                  />
+
+                  <Box
+                    display="flex"
+                    justifyContent="center"
+                    alignItems="center"
+                    flexDirection={{ xs: 'column', s: 'row' }}
```

Comment:
```suggestion
                    flexDirection={{ base: 'column', s: 'row' }}
```

Better to use `base` wherever you're using responsive props. It acts as a default value when value is not defined

Reply:
updated.